src/chart/Pie.js

```diff
@@ -31,7 +31,7 @@
     Pie.prototype._class += " chart_Pie";
     Pie.prototype.implements(I2DChart.prototype);
 
-    Pie.prototype.publish("paletteID", "default", "set", "Palette ID", Pie.prototype._palette.switch());
+    Pie.prototype.publish("paletteID", "default", "set", "Palette ID", Pie.prototype._palette.switch(),{tags:['Basic','Shared']});
```

Comment:
Can this not be calculated on the fly?  Otherwise it will for ever be accidentally wrong (what is it actually used for?)